### ISSUE TITLE
There's no text on my bar - only equalizer animation

### ISSUE THREAD

**Issue by dawidk92** [jakemarsh/atom-spotify#23]
![zrzut ekranu 2015-04-15 o 13 43 30](https://cloud.githubusercontent.com/assets/7023167/7158010/7d61a852-e375-11e4-9675-ceaea3b1ec71.png)

Atom: 0.191.0
atom-spotify: 1.2.0
Spotify: 1.0.3.101.gbfa97dfe
OS X Yosemite 10.10.2

**Comment by pixelpony** [comment]
Same here.

**Comment by STLMikey** [comment]
Same here.

Atom: 0.190.0
Atom-spotify: 1.2.0
Spotify: 1.0.3.101.gbfa97dfe
OSX Yosemite: 10.10.2

**Comment by aaroncraigongithub** [comment]
Me too.

Atom: 0.192.0
Atom-spotify: 1.2.0
Spotify: 1.0.3.101.gbfa97dfe
OSX Yosemite: 10.10.3

**Comment by jaman1020** [comment]
The latest spotify update seems to have broken things for a lot of plugins. I'm not sure but could this be the issue?

**Comment by rsylvian** [comment]
I've nothing. Not even the equalizer animation :(

**Comment by codepuncher** [comment]
Yeah, Spotify has changed a lot over the last couple versions AFAIK.
I have been using Toastify to give me notification toasts about song changes and they also ran into a problem due to Spotify changing how song info is displayed.

They ended up having to have some sort of localserver that could read the contents of Spotify or something... main thing I know is that it uses "chromedriver.exe" for this.

Hope that'll be of some use to someone trying to figure this out.

**Comment by tm1000** [comment]
This library depends on a submodule in node called "spotify-node-applescript". There were some bug fixes made between 2.2.1 (which is what this uses) and 2.2.2. To get the new library (which works) go into atom-spotify and update.

eg

```
cd ~/.atom/packages/atom-spotify
MacBook-Pro:atom-spotify $ npm update
spotify-node-applescript@0.2.2 node_modules/spotify-node-applescript
└── applescript@1.0.0
```
